app/utils/requests/surveyQuestions/mappings.ts

```
@@ -31,11 +30,13 @@ export const mapToSurveyQuestion = (
 export const mapToBasicSurveyQuestion = (
   surveyQuestionData: ResultOf<typeof SurveyQuestionFragment>,
 ): BasicSurveyQuestion => {
-  const project = surveyQuestionData.project as ResultOf<typeof ProjectFragment>;
+  const project = surveyQuestionData.project;
+  const projectId = project?.documentId;
+
   return {
     id: surveyQuestionData.documentId,
     question: surveyQuestionData.question,
-    projectId: project.documentId,
+    projectId: projectId ?? '',
```

should we throw an error instead?